### DISCUSSION TITLE
feature-request: Standard Notes CLI

### DISCUSSION THREAD

**Discussion by Indyandie** [standardnotes/forum#2805] [Suggestions]
Hello team,

I am trying to export all my **Day One** entries into **Standard Notes**. A **Standard Notes** command line tool would be really helpful for migrating a bunch of `md` files. And ti would be really cool to create and update notes from the terminal.  Maybe there is a tool already?

**Comment by johnatpackawhallop** [comment]
There is a CLI tool for working with SN which I just found today, but it seems to be no longer compatible.

https://github.com/jonhadfield/sn-cli

Would love to see this updated or, better still, would love to see an official one from the SN team. I manage some Debian servers via SSH and it would be great to have notes from SN available in that environment.

**Reply by cactus7as** [reply]
Yes, I'd love my agent to take notes in SN. Also use the calendar and other Proton Features from a CLI. CLIs are the future :)
